structure/views.py

```diff
@@ -506,31 +507,53 @@ class StructureStatistics(TemplateView):
 	So not ready that EA wanted to publish it.
 	"""
 	template_name = 'structure_statistics.html'
+	origin = 'structure'
 
 	def get_context_data (self, **kwargs):
 		context = super().get_context_data(**kwargs)
-
 		families = ProteinFamily.objects.all()
 		lookup = {}
 		for f in families:
 			lookup[f.slug] = f.name
 
 		all_structs = Structure.objects.all().prefetch_related('protein_conformation__protein__family')
+		noncomplex_gprots = SignprotStructure.objects.filter(protein__family__slug__startswith='100')
 		all_complexes = all_structs.exclude(ligands=None)
-
 		all_gprots = all_structs.filter(id__in=SignprotComplex.objects.filter(protein__family__slug__startswith='100').values_list("structure__id", flat=True))
+		###### these are query sets for G-Prot Structure Statistics
+		all_g_A_complexes = all_gprots.filter(protein_conformation__protein__family__slug__startswith='001')

```

Comment:
Not for now, but it would be nice if this could be a bit more concise

Reply:
I will mark it as a ToDo while working on the upcoming tweaks